libs/barista-components/filter-field/src/filter-field-multi-select/filter-field-multi-select-trigger.ts

```diff
@@ -80,7 +85,7 @@ export class DtFilterFieldMultiSelectTrigger<T>
     protected _viewportRuler: ViewportRuler,
     protected _platform: Platform,
     protected _overlayContainer: OverlayContainer,
-    _zone: NgZone,
+    protected _zone: NgZone,
```

Comment:
Sorry, not quite what I meant in the previous review. 
The zone does not need to be a member, because you are simply passing it to the super constructor. 

it's fine either way though.

Reply:
Yes, I know you meant to remove the underscore but as all the others had this format I preferred adding the `protected`